public/pages/Rules/components/RuleViewerFlyout/RuleViewerFlyout.tsx

```diff
@@ -78,7 +79,7 @@ export const RuleViewerFlyout: React.FC<RuleViewerFlyoutProps> = ({
   };
 
   const onDeleteRuleConfirmed = async () => {
-    if (!ruleService) {
+    if (!ruleService || !notifications) {
```

Comment:
I don't think, not having the notifications should be a blocker here, we should just not be able to show the notification

Reply:
fixed